include/seqan3/search/algorithm/search.hpp

```diff
@@ -28,14 +28,53 @@ namespace seqan3
 
 /*!\brief Search a query or a range of queries in an index.
  * \tparam index_t    Must model seqan3::FmIndex.
- * \tparam queries_t  Must model std::ranges::RandomAccessRange over the index's alphabet.
- *                    a range of queries must additionally model std::ranges::ForwardRange.
+ * \tparam queries_t  Must model std::ranges::RandomAccessRange over the index's alphabet and std::ranges::SizedRange.
+ *                    A range of queries must additionally model std::ranges::ForwardRange and std::ranges::SizedRange.
  * \param[in] queries A single query or a range of queries.
  * \param[in] index   String index to be searched.
  * \param[in] cfg     A configuration object specifying the search parameters (e.g. number of errors, error types,
  *                    output format, etc.).
- * \returns An object modelling std::ranges::Range containing the hits (the type depends on the specification
-            in `cfg`), or `void` if an on_hit delegate has been specified.
+ * \returns
+ * <table>
+ *   <tr>
+ *     <th>seqan3::text_layout</th>
+ *     <th>seqan3::search_cfg::output</th>
+ *     <th>Result of seqan3::search()</th>
+ *   </tr>
+ *   <tr>
+ *     <td style="text-align:center">\ref seqan3::text_layout "single"</td>
+ *     <td style="text-align:center">\ref seqan3::search_cfg::text_position "text_position"</td>
+ *     <td>A `std::vector<size_t>` representing text positions where the search was successful.</td>
+ *   </tr>
+ *   <tr>
+ *     <td style="text-align:center">\ref seqan3::text_layout "single"</td>
+ *     <td style="text-align:center">\ref seqan3::search_cfg::index_cursor "index_cursor"</td>
+ *     <td>A `std::vector<typename index_t::cursor_type>` containing index_cursors at the text positions where the
+ *         search was successful.</td>
+ *   </tr>
+ *   <tr>
+ *     <td style="text-align:center">\ref seqan3::text_layout "collection"</td>
+ *     <td style="text-align:center">\ref seqan3::search_cfg::text_position "text_position"</td>
+ *     <td>A `std::vector<std::pair<size_t, size_t>>` where the first element of the
+ *         `std::pair` specifies the text index in the collection and the second element contains the position in that
+ *         text of the collection.</td>
+ *   </tr>
+ *   <tr>
+ *     <td style="text-align:center">\ref seqan3::text_layout "collection"</td>
+ *     <td style="text-align:center">\ref seqan3::search_cfg::index_cursor "index_cursor"</td>
+ *     <td>A `std::vector<typename index_t::cursor_type>` containing index_cursors at the text positions where the
+ *         search was successful.</td>
+ *   </tr>
```

Comment:
And the markdown in combination with `<br>` did not work?

Reply:
`<br>` Only inserts a linebreak in the cell content, I still have to put the whole row on one line.